// jest-discordjs-mocks/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.MockWebhookClient = exports.MockWebhook = exports.MockVoiceStateManager = exports.MockVoiceState = exports.MockVoiceRegion = exports.MockVoiceChannel = exports.MockUserManager = exports.MockUser = exports.MockTextChannel = exports.MockTeamMember = exports.MockTeam = exports.MockStreamDispatcher = exports.MockStoreChannel = exports.MockRoleManager = exports.MockReactionUserManager = exports.MockReactionManager = exports.MockReactionEmoji = exports.MockPresenceManager = exports.MockPresence = exports.MockPartialGroupDMChannel = exports.MockNewsChannel = exports.MockMessageReaction = exports.MockMessageManager = exports.MockMessageAttachment = exports.MockMessage = exports.MockInvite = exports.MockIntegration = exports.MockGuildPreviewEmoji = exports.MockGuildPreview = exports.MockGuildMemberRoleManager = exports.MockGuildMemberManager = exports.MockGuildMember = exports.MockGuildManager = exports.MockGuildEmojiRoleManager = exports.MockGuildEmojiManager = exports.MockGuildEmoji = exports.MockGuildChannelManager = exports.MockGuildChannel = exports.MockGuildAuditLogEntry = exports.MockGuild = exports.MockEmoji = exports.MockDMChannel = exports.MockClientVoiceManager = exports.MoockClientUser = exports.MockChannelManager = exports.MockChannel = exports.MockCategoryChannel = exports.MockBroadcastDispatcher = exports.MockBase = exports.MockActivity = void 0;
var discord_js_1 = require("discord.js");
exports.MockActivity = discord_js_1.Activity;
exports.MockBase = discord_js_1.Base;
exports.MockBroadcastDispatcher = discord_js_1.BroadcastDispatcher;
exports.MockCategoryChannel = discord_js_1.CategoryChannel;
exports.MockChannel = discord_js_1.Channel;
exports.MockChannelManager = discord_js_1.ChannelManager;
exports.MoockClientUser = discord_js_1.ClientUser;
exports.MockClientVoiceManager = discord_js_1.ClientVoiceManager;
exports.MockDMChannel = discord_js_1.DMChannel;
exports.MockEmoji = discord_js_1.Emoji;
exports.MockGuild = discord_js_1.Guild;
exports.MockGuildAuditLogEntry = discord_js_1.GuildAuditLogsEntry;
exports.MockGuildChannel = discord_js_1.GuildChannel;
exports.MockGuildChannelManager = discord_js_1.GuildChannelManager;
exports.MockGuildEmoji = discord_js_1.GuildEmoji;
exports.MockGuildEmojiManager = discord_js_1.GuildEmojiManager;
exports.MockGuildEmojiRoleManager = discord_js_1.GuildEmojiRoleManager;
exports.MockGuildManager = discord_js_1.GuildManager;
exports.MockGuildMember = discord_js_1.GuildMember;
exports.MockGuildMemberManager = discord_js_1.GuildMemberManager;
exports.MockGuildMemberRoleManager = discord_js_1.GuildMemberRoleManager;
exports.MockGuildPreview = discord_js_1.GuildPreview;
exports.MockGuildPreviewEmoji = discord_js_1.GuildPreviewEmoji;
exports.MockIntegration = discord_js_1.Integration;
exports.MockInvite = discord_js_1.Invite;
exports.MockMessage = discord_js_1.Message;
exports.MockMessageAttachment = discord_js_1.MessageAttachment;
exports.MockMessageManager = discord_js_1.MessageManager;
exports.MockMessageReaction = discord_js_1.MessageReaction;
exports.MockNewsChannel = discord_js_1.NewsChannel;
exports.MockPartialGroupDMChannel = discord_js_1.PartialGroupDMChannel;
exports.MockPresence = discord_js_1.Presence;
exports.MockPresenceManager = discord_js_1.PresenceManager;
exports.MockReactionEmoji = discord_js_1.ReactionEmoji;
exports.MockReactionManager = discord_js_1.ReactionManager;
exports.MockReactionUserManager = discord_js_1.ReactionUserManager;
exports.MockRoleManager = discord_js_1.RoleManager;
exports.MockStoreChannel = discord_js_1.StoreChannel;
exports.MockStreamDispatcher = discord_js_1.StreamDispatcher;
exports.MockTeam = discord_js_1.Team;
exports.MockTeamMember = discord_js_1.TeamMember;
exports.MockTextChannel = discord_js_1.TextChannel;
exports.MockUser = discord_js_1.User;
exports.MockUserManager = discord_js_1.UserManager;
exports.MockVoiceChannel = discord_js_1.VoiceChannel;
exports.MockVoiceRegion = discord_js_1.VoiceRegion;
exports.MockVoiceState = discord_js_1.VoiceState;
exports.MockVoiceStateManager = discord_js_1.VoiceStateManager;
exports.MockWebhook = discord_js_1.Webhook;
exports.MockWebhookClient = discord_js_1.WebhookClient;
